modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ObjectRelationship/ObjectRelationship.tsx

```diff
@@ -324,7 +324,7 @@ export default function ObjectRelationship({
 						}
 					}}
 					placeholder={placeholder}
-					readOnly={readOnly}
+					readOnly={true}
```

Comment:
@Valmir-unicap the test `LocalFile.ObjectRelationships#CanAddOneToManyRelationWithNativeObject` seems to be failing because of this change you made.

As you are always setting this field to be `readOnly`, we can no longer edit this field for any other use case. 

We should **only** do this for the use case described in the ticket, i.e., relationship field with Account Restriction.

Thanks!